src/Faker/Provider/de_CH/Address.php

```diff
@@ -0,0 +1,113 @@
+    );
+
+    protected static $canton = array(
+        array('short' => 'AG', 'name' => 'Aargau'), array('short' => 'AI', 'name' => 'Appenzell Innerrhoden'), array('short' => 'AR', 'name' => 'Appenzell Ausserrhoden'), array('short' => 'BE', 'name' => 'Bern'), array('short' => 'BL', 'name' => 'Basel-Landschaft'), array('short' => 'BS', 'name' => 'Basel-Stadt'), array('short' => 'FR', 'name' => 'Freiburg'), array('short' => 'GE', 'name' => 'Genf'), array('short' => 'GL', 'name' => 'Glarus'), array('short' => 'GR', 'name' => 'Graubünden'), array('short' => 'JU', 'name' => 'Jura',), array('short' => 'LU', 'name' => 'Luzern'), array('short' => 'NE', 'name' => 'Neuenburg'), array('short' => 'NW', 'name' => 'Nidwalden'), array('short' => 'OW', 'name' => 'Obwalden'), array('short' => 'SG', 'name' => 'St. Gallen'), array('short' => 'SH', 'name' => 'Schaffhausen'), array('short' => 'SO', 'name' => 'Solothurn'), array('short' => 'SZ', 'name' => 'Schwyz'), array('short' => 'TG', 'name' => 'Thurgau'), array('short' => 'TI', 'name' => 'Tessin'), array('short' => 'UR', 'name' => 'Uri'), array('short' => 'VD', 'name' => 'Waadt'), array('short' => 'VS', 'name' => 'Wallis'), array('short' => 'ZG', 'name' => 'Zug'), array('short' => 'ZH', 'name' => 'Zürich')
```

Comment:
Please split the line for easier reading. Also, why use the verbose format `short/name` rather than a key/value like 

```php
array(
  'AG' => 'Aargau',
  'AI' => 'Appenzell Innerrhoden',
  // etc
);
```

?